lib/Elastica/Index/Settings.php

```diff
@@ -118,7 +120,18 @@ public function setNumberOfReplicas($replicas)
      */
     public function setReadOnly($readOnly = true)
     {
-        return $this->set(array('blocks.read_only' => $readOnly));
+        return $this->set(array('blocks.write' => $readOnly));
+    }
+
+    /**
+     * getReadOnly
+     *
+     * @access public
+     * @return bool
+     */
+    public function getReadOnly()
+    {
+        return $this->get('blocks.write') === 'true'; // ES returns a string for this setting
```

Comment:
This variable sounds so counterintuitive. But it is 100% correct. I also checked the ES docs.